service/frontend/workflow_handler.go

```diff
@@ -6052,3 +6052,26 @@ func (wh *WorkflowHandler) ListWorkers(
 		NextPageToken: resp.GetNextPageToken(),
 	}, nil
 }
+
+func (wh *WorkflowHandler) UpdateTaskQueueConfig(
+	ctx context.Context, request *workflowservice.UpdateTaskQueueConfigRequest,
+) (*workflowservice.UpdateTaskQueueConfigResponse, error) {
+	if request == nil {
+		return nil, errRequestNotSet
+	}
+	namespaceName := namespace.Name(request.GetNamespace())
+	namespaceID, err := wh.namespaceRegistry.GetNamespaceID(namespaceName)
+	if err != nil {
+		return nil, err
+	}
+	resp, err := wh.matchingClient.UpdateTaskQueueConfig(ctx, &matchingservice.UpdateTaskQueueConfigRequest{
```

Comment:
We should do some more validation here before we hand it off:
- prohibit setting rate limit on workflow task queue (at least until we have a distributed rate limiter that can handle sticky queues)
- prohibit negative values
- check maximum length of `reason` fields and anything else that we're going to store in metadata

But let's do them in a follow-up PR.